src/Main.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Scanner;
+
+public class Main {
+    static Scanner keyboard = new Scanner(System.in);
+
+    public static void chekYear(int year) {
```

Comment:
```suggestion
    public static void printIsLeapYearOrNot(int year) {
```
название метода должно отражать то, что он делает